game/plugins/src/main/kotlin/gg/rsmod/plugins/content/areas/shilovillage/fernahei's.plugin.kts

```diff
@@ -0,0 +1,55 @@
+package gg.rsmod.plugins.content.areas.shilovillage
+
+import gg.rsmod.plugins.content.mechanics.shops.CoinCurrency
+
+/**
+ * @author Alycia <https://github.com/alycii>
```

Comment:
Put your name here.

Reply:
is way auto add my name then jw ? on inteji